symengine/dict.h

```diff
@@ -42,6 +43,7 @@ typedef std::map<RCP<const Integer>, unsigned,
         RCPIntegerKeyLess> map_integer_uint;
 typedef std::map<unsigned, integer_class>
        map_uint_mpz;
+typedef std::map<int, Expression> map_int_Expr;
```

Comment:
`SeriesBase` requires that the coefficient type has arithmetic operators overloaded.